.github/workflows/issues-gh-to-ado.yml

```diff
@@ -0,0 +1,35 @@
+on:
+  workflow_call:
+    inputs:
+      ado_area_path:
+        description: "ADO area path where ADO tickets are created."
+        required: true
+        default: "2DegreesInvesting\\Test Team 1"
```

Comment:
Is this the team you want to use for the action?

Reply:
There is currently no Team set as a default here, and "Test Team 1" seems appropriately ring fenced that we won't cause chaos.

I will test this action out in the wild this sprint, and update the default Team once Matze an I have figured that part out :-)